docs/language_reference.html

```diff
@@ -1020,6 +1020,9 @@ <h3 id="break">break</h3>
 statement inside of them, that exits the enclosing loop. Alternatively,
 use <code>return</code> or <code>return from</code> (see above) for more
 complex cases.</p>
+<h3 id="continue">continue</h3>
```

Comment:
This is a generated file from `docs/source/language_reference.md`, you want to update it there.